features/admin.identity-providers.v1/configs/ui.ts

```diff
@@ -28,6 +28,7 @@ import { ReactComponent as GearsIcon } from "../../themes/default/assets/images/
 import MagicLinkLogo from "../../themes/default/assets/images/icons/magic-link-icon.svg";
 import MagnifierColoredIcon from "../../themes/default/assets/images/icons/magnifier-colored-icon.svg";
 import TOTPIcon from "../../themes/default/assets/images/icons/outline-icons/clock-outline.svg";
+import PushIcon from "../../themes/default/assets/images/icons/push.svg";
```

Comment:
Please move this to either connections or push features module.

Reply:
Similar implementation is already there in the connections. I think this is to show the push authenticator in the add authentication modal in the login flow tab. @ZiyamSanthosh could you please confirm?

Reply:
We are planing to deprecate the identity-providers module. So better to avoid using the files there. Since this is just an icon we can simply move it to push notification feature module.

Reply:
We still use AuthenticatorFormFactory from identity-providers feature to render the authenticator's edit form. Therefore we still need to add these to identity providers module

[1] https://github.com/wso2/identity-apps/blob/master/features/admin.connections.v1/components/edit/edit-multi-factor-authenticator.tsx#L24